pkg/apis/alerting/v1/validate.go

```diff
@@ -505,5 +513,57 @@ func (n *Notification) Validate() error {
 			return validation.Error("if specifying a cluster id property, it must be set")
 		}
 	}
+	if _, ok := n.Properties[NotificationPropertyOpniUuid]; !ok {
+		n.Properties[NotificationPropertyOpniUuid] = util.HashStrings([]string{n.Title, n.Body})
+	}
+	return nil
+}
+
+func (l *ListNotificationRequest) Validate() error {
```

Comment:
Validate() shouldn't have side effects, setting defaults should happen elsewhere